iac-extensions/arm/src/main/resources/org/sonar/l10n/azureresourcemanager/rules/azureresourcemanager/S1192.html

```diff
@@ -0,0 +1,93 @@
+<h2>Why is this an issue?</h2>
+<p>Duplicated string literals make the process of refactoring complex and error-prone, as any change would need to be propagated on all
+occurrences.</p>
+<h3>Exceptions</h3>
+<p>The following are ignored:</p>
+<ul>
+  <li> literals with fewer than 5 characters </li>
+  <li> The <code>apiVersion</code> of a resource (see rule {rule:azureresourcemanager:S6893}) </li>
```

Comment:
This is the old rspec before your merge. Before a release we will update so I don't see an issue here.